pelicanconf.py

```diff
@@ -11,6 +12,14 @@
 FULLURL = "https://heasarc.gsfc.nasa.gov/docs/tess"
 
 
+#Get the latest stats for the planet counter. Note the planet and paper counts are generated with a python script to be added in a future commit (consider them static at the moment)
+PLANETCOUNT = open(PATH+'/statistics/planetcount.txt','r').readline() if os.path.exists(PATH+'/statistics/planetcount.txt') else 0
```

Comment:
Nice error catch!

Reply:
There is not necessarily a benefit to having two txt files, other than I was trying to keep the modules as separate as possible to make it easier to change down the line or use elsewhere. But if we do expand later it would make sense to save everything in the same csv. I can certainly work on implementing this. Is it something I should look into before pushing this, or in a future pull request?